src/axom/inlet/Field.hpp

```diff
@@ -347,6 +348,18 @@ class Field : public std::enable_shared_from_this<Field>
    */
   bool get(std::string& value);
 
+  template <typename T>
```

Comment:
It seems like it might be confusing to have two functions called `get ` that behave differently.
One is of the form:
```cpp
bool get(PrimitiveType& value)
```
the other is of the form:
```
template<typename T>
T get()
```

Should one of them be renamed?
Would `contains` be better for the form that returns the value as a reference out parameter?

Reply:
Should have ready your PR description better :)

Reply:
To avoid ambiguity I have renamed the `get(T& val)` to `get_to`